doc/examples/moveit_configuration/moveit_configuration_tutorial.rst

```diff
@@ -0,0 +1,183 @@
+MoveIt Configuration
+==================================
+
+The recommended way of configuring MoveIt for your robot is by creating a colcon package containing the MoveIt Configuration.
+
+Suppose you would like to create a configuration package for some robot named ``my_robot``.
+To do this, you can create a Colcon package named ``my_robot_moveit_config``, whose structure is as follows:
```

Comment:
```suggestion
To do this, you can create a colcon package named ``my_robot_moveit_config``, whose structure is as follows:
```

![image](https://user-images.githubusercontent.com/10466537/233646938-db534366-5f8f-454d-ba0a-fd89d0f6d70b.png)
Looks like lower case always is the "proper" way to spell colcon